appsec/config.go

```diff
@@ -18,22 +18,34 @@ import (
 
 // Configuration environment variables
 const (
-	envAPISecEnabled    = "DD_EXPERIMENTAL_API_SECURITY_ENABLED"
-	envAPISecSampleRate = "DD_API_SECURITY_REQUEST_SAMPLE_RATE"
-	envObfuscatorKey    = "DD_APPSEC_OBFUSCATION_PARAMETER_KEY_REGEXP"
-	envObfuscatorValue  = "DD_APPSEC_OBFUSCATION_PARAMETER_VALUE_REGEXP"
-	envWafTimeout       = "DD_APPSEC_WAF_TIMEOUT"
-	envTraceRateLimit   = "DD_APPSEC_TRACE_RATE_LIMIT"
-	envRules            = "DD_APPSEC_RULES"
+	// EnvAPISecEnabled is the env var used to enable API Security
+	EnvAPISecEnabled = "DD_EXPERIMENTAL_API_SECURITY_ENABLED"
+	// EnvAPISecSampleRate is the env var used to set the sampling rate of API Security schema extraction
+	EnvAPISecSampleRate = "DD_API_SECURITY_REQUEST_SAMPLE_RATE"
```

Comment:
MAYBE the ship has sailed already (in which case, feel free to ignore this), but I wonder if we should use an environment variable name prefix for all configuration pertaining to some feature... Which in this case would mean this should have been `DD_EXPERIMENTAL_API_SECURITY_REQUEST_SAMPLE_RATE`?

Reply:
It may be too late since implementation is started in several libs. I opened a discussion on the RFC anyway.